### DISCUSSION TITLE
SG2 can't deactivate boot entry for windows boot manager as described in guide

### DISCUSSION THREAD

**Discussion by Michachacha** [linux-surface/linux-surface#1630] [Q&A]
Hi everyone,
i have installed LM as dual boot with Win11 on my Surface Go 2, following [this guide](https://github.com/linux-surface/linux-surface/wiki/Surface-Go-2#dual-booting-linux-and-windows). In order to get the Surface to boot into Grub instead of defaulting to Windows, i am supposed to deactivate the boot order entry in the EFI Boot Manager, like so:

`$ sudo efibootmgr -Ab 0001`

...but it won't do it:

`efibootmgr: Boot entry 1 not found
Could not set active state for Boot0001: No such file or directory`

The entry 0001 DOES exist however in the boot order list:

`$ sudo efibootmgr  
[sudo] Passwort für micha:           
BootCurrent: 0002  
Timeout: 0 seconds  
BootOrder: 0001,0002,2001,2002  

Boot0001* Windows Boot Manager	HD(1,GPT,d5cec136-9d6f-4ef5-b595-c32df7d1fdc1,0x800,0x32000)/File(\EFI\Microsoft\Boot\bootmgfw.efi)57494e444f5753000100000088000000780000004200430044004f0042004a004500430054003d007b00390064006500610038003600320063002d0035006300640064002d0034006500370030002d0061006300630031002d006600330032006200330034003400640034003700390035007d00000064000100000010000000040000007fff0400  

Boot0002* ubuntu	HD(1,GPT,d5cec136-9d6f-4ef5-b595-c32df7d1fdc1,0x800,0x32000)/File(\EFI\ubuntu\shimx64.efi) File(.䍒)  

Boot2001* EFI USB Device	RC  

Boot2002* EFI Network	RC`

Any idea, what i'm doing wrong?
Thank you
Micha

**Comment by Michachacha** [comment]
Am i the first to use the guide and encounter this problem?
Am i in the wrong place for this kind of question?

**Comment by dev-nis** [comment]
Hi, I think I had the same or a similar problem on a Surface Go 3 once. I don't remember exactly what the issue was but I remember that for my case, I needed to download an older version of `efibootmgr`
